### PR TITLE
CDPT-440 mailto link

### PR DESCRIPTION
**Pull request by MeenaModhvadia**
## Description
- Mail feature to email parliamentary questions team from early bird list

### CODE REVIEW

File: app/views/early_bird_dashboard/index.html.slim [hunk 1]
```diff
@@ -62,3 +62,4 @@
                 br
               - else
                 span.constituency #{question.member_constituency}
+            p =link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
```
Comment (vertism) [inline app/views/early_bird_dashboard/index.html.slim:65]: I don't have much experience of slim, but it seems to be standard to have a space between `=` and the content.
You might also want to put the link_to on a separate line.
```suggestion
            p 
              = link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
```

Comment (MeenaModhvadia) [reply app/views/early_bird_dashboard/index.html.slim:65]: Hi Andrew,
Thank you for your feedback.
Previous developer's standard way of writing slim was to have everything on one line and to change only this P would look odd

Comment (vertism) [reply app/views/early_bird_dashboard/index.html.slim:65]: Fair enough, consistency is good!

**Review by vertism — APPROVED** [review]
LGTM :)

**Review by wilson1000-MoJ — APPROVED** [review]
Reviewed commits after Andrews. All looks good.

**Comment by wilson1000-MoJ** [comment]
I've just noticed some syntax queries...

File: app/views/early_bird_dashboard/index.html.slim [hunk 2]
```diff
@@ -62,3 +62,5 @@
                 br
               - else
                 span.constituency #{question.member_constituency}
+            - if feature_propose_ao_enabled?
+              p = link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
```
Comment (vertism) [inline app/views/early_bird_dashboard/index.html.slim:66]: It's very minor, but you could use double quotes for both arguments.

Comment (Polsonby) [reply app/views/early_bird_dashboard/index.html.slim:66]: Thanks, good spot - we'll do that in the next PR 🙏

**Pull request merged**